### ISSUE TITLE
Mallow views

### ISSUE THREAD

**Issue by MaxHalford** [carbonfact/lea#18]
[Mallow](https://github.com/malloydata/malloy) is an expressive query language for data processing. It would we so cool to have it working alongside regular SQL views and Python views. Technically, this would only be a new `MallowView` class in the `views` module. Then, each client could execute the SQL compiled by Malloy.

**Comment by MaxHalford** [comment]
[prql](https://github.com/prql/prql) looks interesting too

**Comment by LeonardoNatale** [comment]
"Malloy currently works with SQL databases BigQuery, Postgres, and querying Parquet and CSV via DuckDB." ([source](https://docs.malloydata.dev/documentation/))

Meaning we wouldn't be able interact with a local database file such as `jaffle_shop.db`

**Comment by MaxHalford** [comment]
But wouldn't it work if we just used Mallow as a translation layer to generate SQL?

**Comment by LeonardoNatale** [comment]
> But wouldn't it work if we just used Malloy as a translation layer to generate SQL?

After playing a bit with their python package, I couldn't manage to just translate from `MALLOY` to `SQL` without having to load a specific source before

**Comment by MaxHalford** [comment]
Right, I see. Difficult to know if this is by design on Mallory's side or not.

Is this blocking? Don't we almost have a client available/open when we parse views?

**Comment by LeonardoNatale** [comment]
It might not be blocking for `GoogleQuery`. But it's blocking for `DuckDB` `MotherDuck` as they just support `CSV` and `Parquet` files through `DuckDB`
